README.md

```diff
@@ -88,6 +88,16 @@ import { Document } from 'react-pdf/dist/entry.webpack';
 
 …and you're all set!
 
+#### Parcel
+
+If you use Parcel, it's not a problem either. Instead of directly importing/requiring `'react-pdf'`, import it like so:
+
+```js
+import { Document } from 'react-pdf/dist/entry.parcel';
```

Comment:
I see `react-pdf/build` in my `node_modules`, not `react-pdf/dist`. Probably, you should change it everywhere in `README.md`. Or I missed something.

Reply:
In 3.0.0, the path to precompiled JS is standard `/dist`.

Reply:
Got it. No worries.